src/backend/base/langflow/components/tavily/tavily_crawl.py

```
@@ -0,0 +1,217 @@
+                payload["instructions"] = self.instructions
+
+            for param in ["select_paths", "select_domains", "exclude_paths", "exclude_domains", "categories"]:
+                value = getattr(self, param, None)
+                if value:
+                    try:
+                        value_list = json.loads(value)
+                        if isinstance(value_list, list):
+                            payload[param] = value_list
+                        else:
+                            payload[param] = [value]
+                    except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)